engine/security/trust/trust_sandbox.md

```diff
@@ -74,65 +74,190 @@ the `trustsandbox` container, the Notary server, and the Registry server.
 
 3. Add the following to the new file.
 
-        version: "2"
+        ```yaml
+        version: "2.1"
         services:
-          notaryserver:
-            image: dockersecurity/notary_autobuilds:server-v0.5.1
+          bootstrap:
+            image: golang:1.13-alpine
+            networks:
+              - mdb
+            depends_on:
+              - mysql
             volumes:
-              - notarycerts:/var/lib/notary/fixtures
+              - ./notary/migrations:/go/migrations
+            environment:
+              GO111MODULE: 'on'
+              MIGRATE_VER: v4.6.2
+            command: |-
+              sh -c 'go get -tags 'mysql' github.com/golang-migrate/migrate/v4/cli@$${MIGRATE_VER} &&
+                mv /go/bin/cli /go/bin/migrate &&
+                SERVICE_NAME=notary_signer ./migrations/migrate.sh &&
+                SERVICE_NAME=notary_server ./migrations/migrate.sh'
+
+          server:
+            image: notary:server
             networks:
-              - sandbox
+              mdb:
+              sig:
+              sandbox:
+                aliases:
+                  - notary-server
+            ports:
+              - "8080"
+              - "4443:4443"
+            volumes:
+              - ./certs:/certs
+            depends_on:
+              - mysql
+              - signer
+              - bootstrap
+            entrypoint: /usr/bin/env sh
+            command: -c 'sleep 21s && notary-server -config server-config.json'
+
+          signer:
+            image: notary:signer
+            networks:
+              mdb:
+              sig:
+                aliases:
+                  - notarysigner
+            volumes:
+              - ./certs:/certs
+            depends_on:
+              - mysql
+              - bootstrap
             environment:
-              - NOTARY_SERVER_STORAGE_TYPE=memory
-              - NOTARY_SERVER_TRUST_SERVICE_TYPE=local
-          sandboxregistry:
-            image: registry:2.4.1
+              NOTARY_SIGNER_DEFAULT_ALIAS: timestamp_1
+              NOTARY_SIGNER_TIMESTAMP_1: testpassword
+            entrypoint: /usr/bin/env sh
+            command: -c 'sleep 20s && notary-signer -config signer-config.json'
+
+          mysql:
```

Comment:
Calls to the mysql database fail because the server and signer try to access the database with a non existent user ("server" and "signer")